pints/toy/_high_dimensional_gaussian.py

```diff
@@ -15,8 +15,10 @@
 
 class HighDimensionalGaussianLogPDF(ToyLogPDF):
     """
-    High-dimensional multivariate Gaussian log pdf, with off-diagonal
-    correlations.
+    High-dimensional zero-mean multivariate Gaussian log pdf, with off-diagonal
+    correlations. Specifically, the covariance matrix Sigma is constructed so
```

Comment:
Maybe we should start a new paragraph at "Specifically ... " as it is quite detailed.

Reply:
Good idea: done.